packages/jaeger-ui/src/components/SearchTracePage/SearchResults/index.js

```diff
@@ -173,20 +210,17 @@ export default class SearchResults extends React.PureComponent<SearchResultsProp
         </div>
         <div>
           {diffSelection}
-          <ul className="ub-list-reset">
-            {traces.map(trace => (
-              <li className="ub-my3" key={trace.traceID}>
-                <ResultItem
-                  durationPercent={getPercentageOfDuration(trace.duration, maxTraceDuration)}
-                  isInDiffCohort={cohortIds.has(trace.traceID)}
-                  linkTo={getLocation(trace.traceID, { fromSearch: searchUrl })}
-                  toggleComparison={this.toggleComparison}
-                  trace={trace}
-                  disableComparision={disableComparisons}
-                />
-              </li>
-            ))}
-          </ul>
+          <ListView
+            dataLength={this.props.traces.length}
+            itemHeightGetter={() => 155}
```

Comment:
I think this needs to have the `initialDraw` prop set, otherwise it will render 300 items, from the start.